database/migrations/2022_19_10_000000_add_twitter_refresh_token.php

```diff
@@ -0,0 +1,22 @@
+<?php
+
+use Illuminate\Database\Migrations\Migration;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Support\Facades\Schema;
+
+return new class extends Migration
+{
+    public function up(): void {
+        Schema::table('social_login_profiles', static function(Blueprint $table) {
+            $table->string('twitter_refresh_token', 2048)->nullable();
```

Comment:
Where comes the number 2048 from? Is this a fixed length, max. length or just estimated?
Maybe use `->text` here?

Reply:
default length wasn't enough and twitter doesn't document it, will change it to text